src/webgpu/api/validation/createBindGroup.spec.ts

```diff
@@ -25,60 +22,69 @@ function clone<T extends GPUTextureDescriptor>(descriptor: T): T {
 
 export const g = makeTestGroup(ValidationTest);
 
-g.test('binding_count_mismatch').fn(async t => {
-  const bindGroupLayout = t.device.createBindGroupLayout({
-    entries: [{ binding: 0, visibility: GPUShaderStage.COMPUTE, buffer: { type: 'storage' } }],
-  });
+g.test('binding_count_mismatch')
+  .desc('Test that the number of entries must match the number of entries in the BindGroupLayout.')
```

Comment:
I think it would be ideal to slightly refactor these two tests (this one and the next one) to parameterize between the control case and bad case. This makes it possible by construction to see exactly how the two cases differ.

Reply:
Done, please make sure that this is what you had in mind.